staking/slash/double-sign.go

```diff
@@ -352,7 +352,8 @@ func delegatorSlashApply(
 				}
 
 				// NOTE Assume did as much as could above, now check the undelegations
-				for _, undelegate := range delegationNow.Undelegations {
+				for i := range delegationNow.Undelegations {
+					undelegate := delegationNow.Undelegations[i]
```

Comment:
this is no-op, since undelegation.Amount is a pointer. But better be explicit on reference.